recipes/gklib/meta.yaml

```
@@ -11,7 +11,8 @@ source:
 
 build:
   number: 0
-  # skip: true  # [win]
+  run_exports:  # [not win]
```

You would also need to apply the selector to the `pin_subpackage` line as well though I'd suggest just removing them instead:
```suggestion
  run_exports:
```
(it will cause there to be a pointless dependency installed but it doesn't really matter and it has the benifit of tracking the ABI still)

---

Ah ok that makes sense - fixed by 38bf7b6.